### DISCUSSION TITLE
v2.0.0.rc1

### DISCUSSION THREAD

**Discussion by matjack1** [omersiar/esp-rfid#656] [General]
The second release candidate for v2.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/esprfid/esp-rfid/releases/tag/'>v2.0.0.rc1</a>.</em>

**Comment by matjack1** [comment]
Hey! I'm going to leave this RC around for at least two weeks. If nothing new pops up, then I will release the long due v2 :)

**Reply by konradhuebner** [reply]
@matjack1 One thing I noticed is that if you browse to the device with Safari on iPhone, the system basic authentication window pops up, asking for a username and password - in addition to the website form field asking for only a password. Besides that, with the ESP-rfid blue board and a Weigand reader, the RC1 works well on two devices. Also updating one, including registered user, worked.

**Reply by konradhuebner** [reply]
oh, and Bountysource seems to be down, so the newly added "donate" button is not working. No idea what's up with them, but maybe you check for another donation site?
